### ISSUE TITLE
invalid preview window option: border-rounded|up 2

### ISSUE THREAD

**Issue by plum** [ConnerWill/yayfzf#3]
Hello,

I  installed fomr  aur (archlinux);

upon issuing the cmmand 'yayfzf'  the following is returned, in failure ot launch;

i```
nvalid preview window option: border-rounded|up
2
```

I'd like to sort out the problem, with a little help, if you can tell me what the error points to.
Thanks,
John

**Comment by ConnerWill** [comment]
@plum 

# FZF Error

What version of *fzf* do you have installed?

It seems to me you have an older version of *fzf* installed that does not support *preview window* options that are in *yayfzf*

You can check the version of *fzf* you have installed by running the following command:

```bash
fzf --version
```

The current version of *fzf* is `0.38.0`

```bash
$  fzf --version
0.38.0 (352ea072)
```
Please make sure you have *fzf* up to date. You can update all packages with the following command *(Assuming you are using yay)*:

```bash
yay -Syu
```

Once you have the latest version of *fzf* installed, try running *yayfzf* again. 

# TLDR

Update *fzf* to the latest version

```bash
yay -Syu
```

**Please let me know if this solves your issue**

**Comment by plum** [comment]
@ ConnerWill 
You're perfectly correct; Thanks for kind reply.  it's fine now, with updated fzf version; I had been running fzf-git , installed from aur with yay (2.x), and now have switched to the more recent version with the pacman community repo  installation. 
This was a case of where the git version was some way behind, for whatever reason, perhaps keeping it bare-boned for a way of  structuring further enhancements?
